backend/api/main.py

```diff
@@ -6,13 +6,17 @@
 import uvicorn
 from fastapi import Depends, FastAPI, HTTPException
 from fastapi.middleware.cors import CORSMiddleware
-from pydantic import BaseModel, Field
+from pydantic import BaseModel, Field, Extra, validator
 from redis import asyncio as aioredis
 import aiohttp
-from settings import settings
```

Comment:
@Bhavdeep13  Did this code line not work for you? I see you added sys.path

Reply:
OK, if it is environment specifc problem, I will remove it from code. The issue is that Pycharm adds the paths automatically for us, and the default VS code does not. But you can check this extension, which should provide similar functionality as Pycharm https://marketplace.visualstudio.com/items?itemName=ms-python.python

Reply:
Or you can manually add the path in your environments